lib/sentry/rescue_evss_errors.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Sentry
+  module RescueEVSSErrors
+    private
+
+    def rescue_evss_errors
+      yield
+    rescue EVSS::ErrorMiddleware::EVSSError => e
+      log_message_to_sentry(e, :warn)
```

Comment:
`log_message_to_sentry` is for logging text not exceptions, you want to use `log_exception_to_sentry`, actually we should wait for this [pr](https://github.com/department-of-veterans-affairs/vets-api/pull/1809) to be merged, i change the `log_exception_to_sentry` method to allow changing the warning level

also sorry i wasn't more specific in the issue, but can you just ignore the `EVSS_15005` error in document upload and the `EVSS_7022` and `EVSS_7021` errors in `ClaimsService#all_claims`? we already reported those errors to evss and we're pretty sure they're not caused by our code. for the other errors we can evaluate them on an individual basis and decide if we should ignore.

Reply:
Ah, I see how I was using `log_message_to_sentry` incorrectly.  Thanks.

However, `log_exception_to_sentry` still captures the exception with Raven. https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9659 says to `rescue these evss errors, log them as level: warning instead of level: error and reraise as SentryIgnoredError (which is ignored by Sentry)`. 
So, to clarify, what do we want to happen to these?
Do we capture exception with Sentry as level: warning and with Rails.logger (i.e. use `log_exception_to_sentry`)
Or, just log it with Rails.logger?
Or something else?
Thanks.

Reply:
use `log_exception_to_sentry` with `level: warning`